src/main/java/net/fabricmc/installer/client/ClientHandler.java

```diff
@@ -52,13 +55,40 @@ public void install() {
 				if (createProfile.isSelected()) {
 					ProfileInstaller.setupProfile(mcPath, profileName, gameVersion);
 				}
+				SwingUtilities.invokeLater(() -> showInstalledMessage(loaderVersion, gameVersion));
 			} catch (Exception e) {
 				error(e);
 			}
 			buttonInstall.setEnabled(true);
 		}).start();
 	}
 
+	private void showInstalledMessage(String loaderVersion, String gameVersion) {
+		JLabel label = new JLabel();
+		Font font = label.getFont();
+		Color color = label.getBackground();
+
+		String style = "font-family:" + font.getFamily() + ";" + "font-weight:" + (font.isBold() ? "bold" : "normal") + ";" +
```

Comment:
Might be worth moving this style thing to a string builder, and another method so it can be re-used by the other similar place.

Reply:
`String.format()` would be better than a builder imo, but yes.